@xen-orchestra/lite/src/components/vm/VmHeader.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <TitleBar :icon="faDisplay">
+    {{ name }}
+    <template #actions>
+      <AppMenu shadow placement="bottom-end">
+        <template #trigger="{ open, isOpen }">
+          <UiButton :active="isOpen" :icon="faPowerOff" @click="open">
+            {{ $t("change-state") }}
+            <UiIcon :icon="faAngleDown" />
+          </UiButton>
+        </template>
+        <MenuItem :disabled="!isHalted" :icon="faPlay">
+          {{ $t("start") }}
+        </MenuItem>
+        <MenuItem :disabled="!isHalted" :icon="faServer">
+          {{ $t("start-on-host") }}
+          <template #submenu>
+            <MenuItem
+              v-for="host in hostStore.allRecords"
+              v-bind:key="host.$ref"
+              :icon="faServer"
+            >
+              <div class="wrapper">
+                {{ host.name_label }}
+                <div>
+                  <UiIcon
+                    :icon="
+                      host.$ref === poolStore.pool?.master ? faStar : undefined
+                    "
+                    class="star"
+                  />
```

Comment:
Not for this PR but maybe we should have some kind of `renderItem` so that we don't have to do that every time, what do you think?